test/bench/helpers.hpp

```diff
@@ -3,21 +3,22 @@
 // SPDX-License-Identifier: Apache-2.0
 #pragma once
 
+#include "test/state/host.hpp"
+#include "test/state/test_state.hpp"
 #include "test/utils/utils.hpp"
 #include <benchmark/benchmark.h>
 #include <evmc/evmc.hpp>
 #include <evmc/mocked_host.hpp>
 #include <evmone/advanced_analysis.hpp>
 #include <evmone/advanced_execution.hpp>
 #include <evmone/baseline.hpp>
-#include <evmone/eof.hpp>
 #include <evmone/vm.hpp>
 
 namespace evmone::test
 {
 extern std::map<std::string_view, evmc::VM> registered_vms;
 
-constexpr auto default_revision = EVMC_ISTANBUL;
+constexpr auto default_revision = EVMC_PRAGUE;
```

Comment:
Maybe don't change this value if not needed. Later we will need to update synthetic tests or remove them in favor of EEST.